go/apps/jsbox/outbound.py

```diff
@@ -246,6 +290,14 @@ def handle_send_to_endpoint(self, api, command):
         if not isinstance(command.get('to_addr'), unicode):
             return self._mkfaild(
                 command, reason=u"'to_addr' must be given in sends.")
+        if not isinstance(command.get('helper_metadata'), (dict, type(None))):
+            return self._mkfaild(
+                command, reason=u"'helper_metadata' must be object or null.")
+        cmd_helper_metadata = command.get('helper_metadata', {})
+        try:
+            self._check_helper_metadata(cmd_helper_metadata)
+        except InvalidHelperMetadata as err:
+            return self._mkfaild(command, reason=unicode(err))
```

Comment:
If we rewrite `_check_helper_metadata(helper_metadata)` as `_get_helper_metadata(command)` or something, we could avoid some of this duplication. Is that a thing worth doing?